master/reference/cni-plugin/configuration.md

```diff
@@ -180,3 +180,55 @@ When using the CNI `host-local` IPAM plugin, a special value `usePodCidr` is all
 ```
 
 When making use of the `usePodCidr` option, the Calico CNI plugin requires read-only Kubernetes API access to the `Nodes` resource.
+
+##### IPAM Manipulation with Kubernetes Annotations
```

Comment:
I think it's best to write this documentation in terms of "what does a user want to do?" rather than "how do they do it?"

So instead of saying something like "Using Kubernetes annotations", it's better to break this into two top level items "Specifying IP Pools" and "Requesting a specific IP address"